### ISSUE TITLE
Create Teams Dark Theme Variables

### ISSUE THREAD

**Issue by levithomason** [microsoft/fluent-ui-react#298]
We need to create dark theme variables for all components.  We will first do this explicitly in siteVariables and each component's variables file.  Afterward, we can consider an abstraction for applying "color schemes" to components.  Let's wait on that until we can see the pattern that emerges.

Please link you PR after the item you are completing below.

### Doc Site

- [x] ComponentExample.tsx background should change on theme switch #316
- [ ] (Low priority) theme switcher for each Component Example

### Components P1
- [x] Button @bcalvery #336
- [ ] Button Contrast @bcalvery #381
- [x] Divider @codepretty #338
- [ ] Header @bcalvery 
- [ ] Icon
- [ ] Image
- [ ] Layout
- [ ] Segment
- [x] Text @codepretty #317 

### Components P2
- [ ] Accordion
- [ ] Attachment
- [x] Avatar @bcalvery #373
- [ ] Chat
- [ ] Grid
- [ ] Input
- [ ] ItemLayout
- [ ] Label
- [ ] List
- [ ] Menu
- [ ] Popup
- [ ] Portal
- [ ] RadioGroup
- [x] Status @bcalvery #373

**Comment by codepretty** [comment]
We are doing Dark theme & Contrast theme at the same time, so it is unnecessary to keep 2 separate issues for this. Merged this issue into #349
